src/runtime/client-config-manager.js

```diff
@@ -112,6 +112,15 @@ export class ClientConfigManager {
     return this.clientOptions_.theme || ClientTheme.LIGHT;
   }
 
+  /**
+   * Returns whether scrolling on main page should be allowed when
+   * subscription or contribution dialog is displayed.
+   * @return {boolean}
+   */
+  shouldAllowScroll() {
```

Comment:
may be worth converting this to `shouldDisableScroll` or (some other negation) to match the config `shouldDisableBodyScrolling`

Reply:
Will leave as `shouldAllowScroll`.